contrib/epee/include/span.h

```diff
@@ -31,6 +31,7 @@
 #include <algorithm>
 #include <cstdint>
 #include <memory>
+#include <string>
```

Comment:
missing include added to fix macOS build. this is required by other Pull Requests as well.